doc/js_tutorials/js_core/js_basic_ops/js_basic_ops.markdown

```diff
@@ -0,0 +1,293 @@
+Basic Operations on Images {#tutorial_js_basic_ops}
+==========================
+
+Goal
+----
+
+Learn to:
+
+-   Access image properties
+-   How to construct Mat
+-   How to copy Mat
+-   How to convert the type of Mat
+-   Access pixel values and modify them
+-   Setting Region of Interest (ROI)
+-   Splitting and Merging images
+
+Accessing Image Properties
+--------------------------
+
+Image properties include number of rows, columns and size, depth, channels, type of image data.
+
+@code{.js}
+let src = cv.imread("canvasInput");
+let rows =  src.cols;
+let columns = src.rows;
+let size = src.size;
+let depth = src.depth();
+let channels = src.channels();
+let type = src.type();
```

Comment:
It's better to use these properties. E.g. add a `console.log('image width ' + src.cols + ' height ' + src.rows ...`